src/FFmpegVideoEncoder.cs

```diff
@@ -338,89 +387,93 @@ public AVFrame MakeFrame(byte* sample, int width, int height)
 
         public byte[]? Encode(AVCodecID codecID, AVFrame avFrame, int fps, bool keyFrame = false)
         {
-            if (!_isDisposed)
+            if (_isDisposed)
+            {
+                return null;
+            }

```

Comment:
This seem redundant if the conditional is a basic type like `bool` and the lock is on a void object and never nulled in the `Dispose` method

```suggestion
```